Lib/test/test_poll.py

```diff
@@ -130,7 +130,7 @@ def test_poll2(self):
         p = proc.stdout
         pollster = select.poll()
         pollster.register( p, select.POLLIN )
-        for tout in (0, 1000, 2000, 4000, 8000, 16000) + (-1,)*10:
+        for tout in (0, 1000, 2000, 4000, 8000, 16000, -1000) + (-1,)*10:
```

Comment:
Style nitpick: I'd move `-1000` to the beginning of the tuple to make it ordered.

Reply:
Ok.